frontend/src/main/scala/bloop/engine/tasks/CompileTask.scala

```diff
@@ -113,6 +112,25 @@ object CompileTask {
           case Left(earlyResultBundle) =>
             compileProjectTracer.terminate()
             Task.now(earlyResultBundle)
+          case Right(CopyResourcesOnly) =>
+            val denylist = Set.empty[Path] // TODO: verify if it shouldn't be computed somehow
```

Comment:
I don't think we have anything that we don't want to copy, this is usually useful for invalidated sources. It's safe to set it to empty for this use case

I wonder if we should delete anything no longer available, but that might be a separate thing altogether.